doc/conf.py

```diff
@@ -12,6 +12,16 @@
 # serve to show the default.
 
 import sys
+from unittest.mock import MagicMock
```

Comment:
I will try to remove on my branch, and see what RtD will do. Then I will report.